soc/arm/silabs_sim3/sim3u/Kconfig.defconfig.series

```diff
@@ -0,0 +1,23 @@
+# SIM3U series configuration options
+
+# Copyright (c) 2023 GARDENA GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_SIM3U
+
+config SOC_SERIES
+	default "sim3u"
+
+config SOC_PART_NUMBER
+	default "SIM3U167-A-GM" if SOC_PART_NUMBER_SIM3U167_A_GM
```

Comment:
Seems to be a way for boards to select the variant in case that information is needed at compile-time. But you're right so I switched it to the B variant which is also selected by our board.